### DISCUSSION TITLE
How to choose the lammps(dp) version for other's trained model

### DISCUSSION THREAD

**Discussion by zs-ustc** [deepmodeling/deepmd-kit#1873] [Q&A]
# Question description
I met this error while using other's trained DP model: 
"version 0.0 in graph, but version 1.0  supported. "

# Detailed information
It's a frozen model in the format of .pb. I've tried different versions of deepmd-kit and lammps(dp) that v2.0.3, v2.0.2, v1.3.3. But I got the same error while I running LAMMPS using it. 

# In general
So how to choose the lammps(dp) version for other's trained model based on this information or the content of .json file? 
I'd be eternally grateful if you can give some advice.

**Comment by HuangJiameng** [comment]
Maybe you can convert your model to a new version with `dp convert-from`.

**Reply by zs-ustc** [reply]
Very much appreciate that , Jiameng. 
I fixed it after several attemps using `dp convert-from` where the origin model version can be quickly ascertained from the last parameter 1.0,1.1,1.2,...
